share/instruments/instruments.xml

```diff
@@ -7489,9 +7489,9 @@
             </Instrument>
             <Instrument id="bells">
                   <family>unpitched-metal-percussion</family>
-                  <trackName>Bells</trackName>
-                  <longName>Bells</longName>
-                  <shortName>Be.</shortName>
+                  <trackName>Ride Bell</trackName>
+                  <longName>Ride Bell</longName>
+                  <shortName>Ri. Be.</shortName>
```

Comment:
@zacjansheski, I changed the shortName from "Be." to "Ri. Be." to match the new long name.